src/include/win.h

```diff
@@ -383,7 +383,11 @@ extern int closedir(DIR *dir);
 
 /* pid related stuff */
 #define WNOHANG 1
+#ifndef LINKS_PYTHON
 #define pid_t	int
+#else
+#include "python.h"
+#endif
```

Comment:
So either python.h will include the pid_t definition or we would define it. So isn't it really available in the earlier files?